smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -105,6 +105,17 @@ function displayReaderList(readers) {
     var indicatorContainer = goog.dom.createDom(
         'span', 'reader-indicator-container', indicator);
 
+    var toolTipText = '';
+    if (reader['status'] == GSC.PcscLiteServer.ReaderStatus.INIT) {
+      toolTipText = 'Initializing card...';
+    } else if (reader['status'] == GSC.PcscLiteServer.ReaderStatus.SUCCESS) {
+      toolTipText = 'Successful card insertion!';
+    } else if (reader['status'] == GSC.PcscLiteServer.ReaderStatus.FAILURE) {
+      toolTipText = 'Failed to connect card!';
+    }
+
+    indicatorContainer.setAttribute('title', toolTipText);
```

Comment:
Could you please implement it in a localizable way, using the "i18n" functionality that we use for other UI elements?

Reply:
will do